backend/src/modules/company/models/company-customer.ts

```diff
@@ -0,0 +1,15 @@
+import { model } from "@medusajs/utils";
```

Comment:
I think we should rename this model to Employee

Reply:
done